README.md

```diff
@@ -17,6 +17,7 @@ Only SSH access is allowed to the bastion host.
   * `ami_id` - AMI ID of Ubuntu (see `samples/ami.tf`)
   * `region` - Region (default, `eu-west-1`)
   * `iam_instance_profile` - IAM instance profile which is allowed to access S3 bucket (see `samples/iam_s3_readonly.tf`)
+  * `detailed_monitoring` - Whether to enable detailed monitoring (default, `true`)
```

Comment:
In order to make it more consistent can we call it `enable_monitoring` instead of `detailed_monitoring`? The rest looks good to me.